Godeps/Godeps.json

```diff
@@ -0,0 +1,24 @@
+{
+	"ImportPath": "github.com/Clever/cron-admin",
+	"GoVersion": "go1.5.1",
```

Comment:
can you re-generate this with `make vendor` so that there's a `Packages` listing?